nursery/terminate-process-by-name.yml

```diff
@@ -0,0 +1,14 @@
+rule:
+  meta:
+    name: terminate process by name
+    namespace: host-interaction/process/terminate
+    author: william.ballenthin@fireeye.com
+    scope: function
+    examples:
+      # - unpacked Cl0p ransomware
+  features:
+    - and:
+      - match: terminate process
+      - match: enumerate processes
+      - or:
+        - offset: 0x24 = pe.szExeFile (x32)

```

Comment:
nice!

Reply:
i wonder if we should have a feature/characteristic that allows us to switch based on the bitness of the file. something like:

```yml
- arch:
  - x32:
    - offset: 0x24 = pe.szExeFile
  - x64:
    - offset: 0x?? = pe.szExeFile
```

Reply:
or maybe make the bitness part of the offset feature?

```yml
- or:
  - offset/32: 0x24 = pe.szExeFile
  - offset/64: 0x?? = pe.szExeFile
```

this would be an easy add, since we could just emit an additional feature for each offset that's tagged with the bitness.

Reply:
https://github.com/fireeye/capa/issues/210